<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CustomizeDlg</class>
 <widget class="QDialog" name="CustomizeDlg">
  <property name="windowModality">
   <enum>Qt::WindowModality::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>300</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>400</width>
    <height>300</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Customize Linver</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>false</bool>
  </property>
  <property name="modal">
   <bool>false</bool>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>6</x>
     <y>6</y>
     <width>388</width>
     <height>256</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="window">
    <attribute name="title">
     <string>Window</string>
    </attribute>
    <widget class="QWidget" name="verticalLayoutWidget">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>378</width>
       <height>222</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="spacing">
       <number>2</number>
      </property>
      <property name="leftMargin">
       <number>4</number>
      </property>
      <property name="topMargin">
       <number>4</number>
      </property>
      <property name="rightMargin">
       <number>4</number>
      </property>
      <property name="bottomMargin">
       <number>4</number>
      </property>
      <item>
       <widget class="QGroupBox" name="groupBox">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>80</height>
         </size>
        </property>
        <property name="title">
         <string>Size</string>
        </property>
        <property name="flat">
         <bool>false</bool>
        </property>
        <property name="checkable">
         <bool>false</bool>
        </property>
        <widget class="QWidget" name="verticalLayoutWidget_4">
         <property name="geometry">
          <rect>
           <x>6</x>
           <y>14</y>
           <width>358</width>
           <height>58</height>
          </rect>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_6">
          <property name="spacing">
           <number>2</number>
          </property>
          <property name="leftMargin">
           <number>2</number>
          </property>
          <property name="topMargin">
           <number>2</number>
          </property>
          <property name="rightMargin">
           <number>2</number>
          </property>
          <property name="bottomMargin">
           <number>2</number>
          </property>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,0">
            <item>
             <widget class="QLabel" name="label">
              <property name="text">
               <string>Width:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSpinBox" name="widthBox">
              <property name="suffix">
               <string extracomment="Maximum: 500"/>
              </property>
              <property name="minimum">
               <number>413</number>
              </property>
              <property name="maximum">
               <number>500</number>
              </property>
              <property name="singleStep">
               <number>5</number>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="1,0">
            <item>
             <widget class="QLabel" name="label_2">
              <property name="text">
               <string>Height:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSpinBox" name="heightBox">
              <property name="suffix">
               <string extracomment="Maximum: 500"/>
              </property>
              <property name="minimum">
               <number>321</number>
              </property>
              <property name="maximum">
               <number>500</number>
              </property>
              <property name="singleStep">
               <number>5</number>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox_2">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>79</height>
         </size>
        </property>
        <property name="title">
         <string>Caption</string>
        </property>
        <property name="flat">
         <bool>false</bool>
        </property>
        <property name="checkable">
         <bool>false</bool>
        </property>
        <widget class="QWidget" name="verticalLayoutWidget_5">
         <property name="geometry">
          <rect>
           <x>6</x>
           <y>14</y>
           <width>358</width>
           <height>60</height>
          </rect>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_7">
          <property name="spacing">
           <number>2</number>
          </property>
          <property name="leftMargin">
           <number>2</number>
          </property>
          <property name="topMargin">
           <number>2</number>
          </property>
          <property name="rightMargin">
           <number>2</number>
          </property>
          <property name="bottomMargin">
           <number>2</number>
          </property>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="1,1">
            <item>
             <widget class="QLabel" name="label_3">
              <property name="text">
               <string>Icon path:</string>
              </property>
             </widget>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_5">
              <property name="spacing">
               <number>2</number>
              </property>
              <item>
               <widget class="QLineEdit" name="iconField"/>
              </item>
              <item>
               <widget class="QPushButton" name="fileBtn">
                <property name="minimumSize">
                 <size>
                  <width>22</width>
                  <height>0</height>
                 </size>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>22</width>
                  <height>16777215</height>
                 </size>
                </property>
                <property name="text">
                 <string/>
                </property>
                <property name="icon">
                 <iconset theme="QIcon::ThemeIcon::DocumentOpen"/>
                </property>
                <property name="autoDefault">
                 <bool>false</bool>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_4" stretch="1,1">
            <item>
             <widget class="QLabel" name="label_4">
              <property name="text">
               <string>Title:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="titleField"/>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Orientation::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QWidget" name="branding">
    <attribute name="title">
     <string>Branding</string>
    </attribute>
    <widget class="QWidget" name="verticalLayoutWidget_6">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>378</width>
       <height>222</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_8">
      <property name="spacing">
       <number>2</number>
      </property>
      <property name="leftMargin">
       <number>4</number>
      </property>
      <property name="topMargin">
       <number>4</number>
      </property>
      <property name="rightMargin">
       <number>4</number>
      </property>
      <property name="bottomMargin">
       <number>4</number>
      </property>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_6" stretch="1,1">
        <item>
         <widget class="QLabel" name="label_6">
          <property name="text">
           <string>Branding image path:</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_7">
          <property name="spacing">
           <number>2</number>
          </property>
          <item>
           <widget class="QLineEdit" name="brandingField"/>
          </item>
          <item>
           <widget class="QPushButton" name="fileBtn2">
            <property name="minimumSize">
             <size>
              <width>22</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>22</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset theme="QIcon::ThemeIcon::DocumentOpen"/>
            </property>
            <property name="autoDefault">
             <bool>false</bool>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QGroupBox" name="brdDetails">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>115</height>
         </size>
        </property>
        <property name="title">
         <string>Details</string>
        </property>
        <property name="flat">
         <bool>false</bool>
        </property>
        <property name="checkable">
         <bool>false</bool>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
        <widget class="QWidget" name="verticalLayoutWidget_7">
         <property name="geometry">
          <rect>
           <x>6</x>
           <y>14</y>
           <width>359</width>
           <height>94</height>
          </rect>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_9">
          <property name="spacing">
           <number>2</number>
          </property>
          <property name="leftMargin">
           <number>2</number>
          </property>
          <property name="topMargin">
           <number>2</number>
          </property>
          <property name="rightMargin">
           <number>2</number>
          </property>
          <property name="bottomMargin">
           <number>2</number>
          </property>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_8" stretch="1,1">
            <item>
             <widget class="QLabel" name="label_7">
              <property name="text">
               <string>Product name:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="productField"/>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_9" stretch="1,1">
            <item>
             <widget class="QLabel" name="label_8">
              <property name="text">
               <string>Copyright:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="copyrightField"/>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_10" stretch="1,1">
            <item>
             <widget class="QLabel" name="label_9">
              <property name="text">
               <string>OS name:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="osField"/>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer_4">
        <property name="orientation">
         <enum>Qt::Orientation::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QWidget" name="details">
    <attribute name="title">
     <string>Details</string>
    </attribute>
    <widget class="QWidget" name="verticalLayoutWidget_2">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>378</width>
       <height>222</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <property name="spacing">
       <number>2</number>
      </property>
      <property name="leftMargin">
       <number>4</number>
      </property>
      <property name="topMargin">
       <number>4</number>
      </property>
      <property name="rightMargin">
       <number>4</number>
      </property>
      <property name="bottomMargin">
       <number>4</number>
      </property>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_15" stretch="1,1">
        <item>
         <widget class="QLabel" name="label_14">
          <property name="text">
           <string>Version:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="versionField"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_14" stretch="1,1">
        <item>
         <widget class="QLabel" name="label_13">
          <property name="text">
           <string>License (RichText):</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="licenseField1"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_16" stretch="1,1">
        <item>
         <widget class="QLabel" name="label_15">
          <property name="text">
           <string>Memory available text:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="memoryField"/>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="verticalSpacer_2">
        <property name="orientation">
         <enum>Qt::Orientation::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QWidget" name="tweaks">
    <attribute name="title">
     <string>Tweaks</string>
    </attribute>
    <widget class="QWidget" name="verticalLayoutWidget_3">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>378</width>
       <height>222</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_5">
      <property name="spacing">
       <number>2</number>
      </property>
      <property name="leftMargin">
       <number>4</number>
      </property>
      <property name="topMargin">
       <number>4</number>
      </property>
      <property name="rightMargin">
       <number>4</number>
      </property>
      <property name="bottomMargin">
       <number>4</number>
      </property>
      <item>
       <widget class="QGroupBox" name="groupBox_4">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>87</height>
         </size>
        </property>
        <property name="title">
         <string>Misc</string>
        </property>
        <property name="flat">
         <bool>false</bool>
        </property>
        <property name="checkable">
         <bool>false</bool>
        </property>
        <widget class="QWidget" name="verticalLayoutWidget_8">
         <property name="geometry">
          <rect>
           <x>6</x>
           <y>12</y>
           <width>359</width>
           <height>68</height>
          </rect>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_10">
          <property name="spacing">
           <number>2</number>
          </property>
          <property name="leftMargin">
           <number>2</number>
          </property>
          <property name="topMargin">
           <number>2</number>
          </property>
          <property name="rightMargin">
           <number>2</number>
          </property>
          <property name="bottomMargin">
           <number>2</number>
          </property>
          <item>
           <widget class="QCheckBox" name="brandingSepCheckBox">
            <property name="text">
             <string>Show branding separator</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="osCheckBox">
            <property name="text">
             <string>Show operating system text</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="memoryCheckBox">
            <property name="text">
             <string>Show memory available</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox_5">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>77</height>
         </size>
        </property>
        <property name="title">
         <string>Details spacers</string>
        </property>
        <property name="flat">
         <bool>false</bool>
        </property>
        <property name="checkable">
         <bool>false</bool>
        </property>
        <widget class="QWidget" name="verticalLayoutWidget_9">
         <property name="geometry">
          <rect>
           <x>6</x>
           <y>12</y>
           <width>359</width>
           <height>58</height>
          </rect>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_11">
          <property name="spacing">
           <number>2</number>
          </property>
          <property name="leftMargin">
           <number>2</number>
          </property>
          <property name="topMargin">
           <number>2</number>
          </property>
          <property name="rightMargin">
           <number>2</number>
          </property>
          <property name="bottomMargin">
           <number>2</number>
          </property>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_17" stretch="1,0">
            <item>
             <widget class="QLabel" name="label_5">
              <property name="text">
               <string>Left:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSpinBox" name="spacerLeftWidth">
              <property name="suffix">
               <string extracomment="Maximum: 64"/>
              </property>
              <property name="minimum">
               <number>0</number>
              </property>
              <property name="maximum">
               <number>64</number>
              </property>
              <property name="singleStep">
               <number>8</number>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_18" stretch="1,0">
            <item>
             <widget class="QLabel" name="label_16">
              <property name="text">
               <string>Right:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSpinBox" name="spacerRightWidth">
              <property name="suffix">
               <string extracomment="Maximum: 64"/>
              </property>
              <property name="minimum">
               <number>0</number>
              </property>
              <property name="maximum">
               <number>64</number>
              </property>
              <property name="singleStep">
               <number>8</number>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer_3">
        <property name="orientation">
         <enum>Qt::Orientation::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </widget>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget_4">
   <property name="geometry">
    <rect>
     <x>6</x>
     <y>268</y>
     <width>388</width>
     <height>26</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="dialogBtns">
    <item>
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Orientation::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QPushButton" name="okBtn">
      <property name="text">
       <string>OK</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="applyBtn">
      <property name="text">
       <string>Apply</string>
      </property>
      <property name="autoDefault">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="cancelBtn">
      <property name="text">
       <string>Cancel</string>
      </property>
      <property name="autoDefault">
       <bool>false</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
